cryptomator/src/main/java/ch/cyberduck/core/cryptomator/CryptoVault.java

```diff
@@ -273,6 +273,7 @@ private void unlock(final VaultConfig vaultConfig, final String passphrase, fina
                     bookmark, LocaleFactory.localizedString("Unlock Vault", "Cryptomator"),
                     message,
                     new LoginOptions()
+                            .keychain(preferences.getBoolean("cryptomator.keychain"))
```

Comment:
We already have "vault.keychain"[^1]
[^1]: https://github.com/iterate-ch/cyberduck/blob/465be7018aeeca4dc2bf6161d9ce83824460004c/cryptomator/src/main/java/ch/cyberduck/core/cryptomator/CryptoVault.java#LL286C75-L286C75